### ISSUE TITLE
Error (C2-12828-1)

### ISSUE THREAD

**Issue by Anorak4213** [ywnico/vitaki-fork#1]
When I select my ps5 console while it is turned on and it is blue, it gives me the error code (C2-12828-1) every single time. I have tried 12 times and nothing worked. 

I am also Odd cauliflour on reddit, you commented on my post earlier and asked me to try this, I am very enthusiastic about this project!

**Comment by ywnico** [comment]
Thanks for making an issue. I haven't encountered this bug.

1) When you clicked on the console, did it ask you for a registration code or was the crash before that? And did a black connection log screen appear, or just a crash immediately on click?

2) Have you installed Chiaki on vita before? If so, can you try renaming the config file (rename `ux0:data/chiaki-vita/chiaki.toml` to `chiaki.toml.bak` or something)

**Comment by niljub** [comment]
I encountered a similar issue with my browser. The solution that worked for me was to delete the application data and restart my Vita. The error message indicates an issue on the application’s side, so it might just be a configuration error.

**Comment by ywnico** [comment]
Good idea, @niljub. Uninstalling vitaki, deleting `chiaki.toml` (or the whole `ux0:data/chiaki-vita` directory), restarting, and then reinstalling it from the vpk is worth a try.

Since I haven't encountered this error and don't have more information, I can't do more debugging than that right now.

**Comment by GuillermoAVeces** [comment]
Hello! I was having the same problem. Tried reinstalling as suggested above and it persisted. So I googled the code and it seems that it is an error that you can get with other game or apps too. Apparently, it is related to some plugins not being compatible with the app. In my case, it was rerescaler, so I uninstalled it and now vitaki is working perfectly.

**Comment by ywnico** [comment]
Nice, thanks for investigating, @GuillermoAVeces! I'll add a note to the readme.

It looks like there's an app blocklist hardcoded into reRescaler (https://github.com/Rinnegatamante/reRescaler/blob/5b33c70cef6078c4d332e5522092dbaae1ef6610/main.c#L589), so I'm not sure whether it's possible to disable it from our end.

**Comment by CavanDuck** [comment]
I'm experiencing the same issue. After registering the console using the pair device code, then choosing console in the menu, as soon as you press X i get the error code. Reinstalling + restarting did not work & renaming the config file didn't work. I do not have rescaler but could be something else.

**Comment by ywnico** [comment]
@CavanDuck Hmm...do you have any other plugins installed?

**Comment by CavanDuck** [comment]
> [@CavanDuck](https://github.com/CavanDuck) Hmm...do you have any other plugins installed?

Have not used my vita In a while, here are all my apps.

![Image](https://github.com/user-attachments/assets/1d32b121-09ac-49ec-97e7-319b1f22da63)

**Comment by CavanDuck** [comment]
Here are plug ins showed in Auto plugin.

![Image](https://github.com/user-attachments/assets/649b5bae-b528-4a04-902c-3e847bf4ddae)

**Comment by ywnico** [comment]
I see, thanks for the info. I guess the most suspicious one is lolicon (and not just because of the name haha), I guess it's possible that the overclocking is causing some issue.

If that's not it, I'll have to have a deeper look later. I also need to implement debug log output so these issues are easier to assess....

I'm traveling without my vita for a few months, so I'm afraid I won't get around to this until I get back.

**Comment by CavanDuck** [comment]
Sadly uninstalling lolicon doesn't seem to make a difference. No worries though, thanks for the suggestion. Safe travels!

Edit: Holding L to disbale plugins on startup seemed to fix the issue, so its definetely one of them lol

Edit 2: I'm dumb, there were more tabs of plugins in auto plugin, it was rescaler in the 'All' tab

**Comment by ywnico** [comment]
> it was rescaler

Nice, glad it was an easy fix after all!

(and I'm also too used to modern UIs and get tripped up a lot on the vita lol)
